spec-tests.Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+node {
+    checkout scm;
+    properties([
+            parameters([
+                stringParam(
+                    description: 'stable tag of binaris/binaris to use for testing',
+                    name: 'tag'
+                    ),
+                stringParam(
+                    description: 'realm',
+                    name: 'realm'
+                    ),
+            ])
+    ]);
+    if (!params.tag) { error('tag not defined'); }
+    if (!params.realm) { error('realm not defined'); }
+    currentBuild.description = "${params.realm} ${params.tag}"
+
+    ansiColor('xterm') {
+        def realm = params.realm;
+        def tag = params.tag;
+
+        timeout(time: 10, unit: 'MINUTES') {
```

Comment:
I'd rather have each of the subjobs have its own timeout.  It doesn't offer the flexibility that allocating a bunch of time to the entire process -- but I would argue that flexibility isn't needed.  OTOH, timeouts (only) on the subjobs mean it's easier to reconfigure a job when we know it's faster/slower.